test/support/rails_app.rb

```diff
@@ -6,6 +6,8 @@ module ActiveModelSerializers
       config.active_support.test_order = :random
       config.action_controller.perform_caching = true
       config.action_controller.cache_store = :memory_store
+
+      config.filter_parameters += [:password]
```

Comment:
Yes, they will fail with the message above. I'm not sure how to change this setting for a single test to write a failure case.